test/e2e/kube_metrics_adapter_test.go

```diff
@@ -71,40 +71,6 @@ var _ = describe("[HPA] Horizontal pod autoscaling (scale resource: Custom Metri
 
 	})
 
-	It("should scale down with Custom Metric of type Object from Skipper [Ingress] [CustomMetricsAutoscaling] [Zalando]", func() {
```

Comment:
Did this test fail? If so we need to verify that no HPAs has this defined in our clusters. (I thought it would still work with this update)